pkg/kubelet/kubelet.go

```diff
@@ -2366,6 +2371,9 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		if !networkConfigured {
 			messages = append(reasons, "network not configured correctly")
 		}
+		if !selinuxConfigured {
+			messages = append(reasons, "SELinux is required by the cluster but not enabled on this node")
```

Comment:
ACK. This means:

* Admins have to decide if a cluster with SELinux or not at the cluster creation time. If they want a different configuration, they have to create a new one.
* If master is configured with SELinux, all Kubelets should configured with SELinux; any Kubelet without such config should be rejected by master by marking as NotReady.
* If master is configured withoutLinux, all Kubelets shouldn't configured with SELinux; any Kubelet with such config should be rejected by master by marking as NotReady. 

In this case, why we need an extra flag in Kubelet here? Why not introducing such to NodeSpec?